901-process.md

```diff
@@ -15,6 +15,38 @@ For example, one of the specifications in this repository is the CNAB Core speci
+
+- Major releases (`X`): Major releases contain breaking changes, including features, fixes, and reorganizations. Implementors should not assume that two major versions are compatible. For example, `1.9.9` is not to be considered compatible with `2.0.0`.
+- Minor releases (`Y`): Minor releases contain features and fixes only. A feature MUST NOT remove or modify existing pieces of the spec (including schemata and file system layouts), but MAY add new things. A minor release SHOULD be backward compatible, though certain security concerns may override this requirement.
+    - Proposed changes to the spec SHOULD be rejected if accepting them will lead to changes in the interpretation of a bundle. (e.g. repurposing an existing field in the `bundle.json` is not allowed by these compatibility rules)
+    - A runtime SHOULD be able to run bundles with the same major version and different minor versions.
+    - A runtime MAY behave differently when processing a package of a newer minor release version, provided that difference is due to new features of the specification.
+    - Bundle authors SHOULD strive to make bundles compatible with all minor versions of the same major version, though they MAY take advantage of additive features.
+- Patch releases (`Z`): Patch release contain fixes to the text of the specification. Patch releases MUST NOT change the behavior of the specification (except in cases where the specification was too vague and the patch clarifies).
+    - Patch releases MUST be both forward and backward compatible to the minor version number
+    - Patch releases MUST NOT make the schema harder to validate against (though they may relax the schema).
+    - Our intention is that CNAB Runtimes SHOULD NOT have to specify behavior at the patch level, as all patch levels are compatible.
+
+Stability markers provide a way to indicate that a bundle or runtime is experimenting with features or fixes. _If an object is tagged with a stability marker, it MUST be treated as incompatible with any other version number. E.g. `1.0.0-GA` MUST be considered incompatible with `1.0.0`. Production artifacts SHOULD NOT use stability markers.
+
+A small number of stability markers are allowed, as determined by [the foundation's governance documents](https://github.com/cnabio/community/blob/master/governance.md):
+
+- `PD`: Pre-draft indicates that the version of the resource is an unstable testing version.
+- `DRAFT`: Draft indicates that the version is an unstable in-development version
+- `GA`: Working Group Approval indicates that this version has been accepted by the maintainers, but not accepted by the Executive Directors. In practice, this marker MUST only be used internally for testing, and never for production. 
+
```

Comment:
This is a bit unclear - the linked governance document contains more phases than the three stability markers described here - are all phases valid markers in versions?

Reply:
No -- only the ones here. I will clarify that.

Reply:
Fixed.